public/app/plugins/panel/nodeGraph/suggestions.ts

```diff
@@ -0,0 +1,32 @@
+import { VisualizationSuggestionsBuilder, VisualizationSuggestionScore } from '@grafana/data';
+import { SuggestionName } from 'app/types/suggestions';
+
+export class NodeGraphSuggestionsSupplier {
+  getListWithDefaults(builder: VisualizationSuggestionsBuilder) {
+    return builder.getListAppender<{}, {}>({
+      name: SuggestionName.NodeGraph,
+      pluginId: 'nodeGraph',
+    });
+  }
+
+  getSuggestionsForData(builder: VisualizationSuggestionsBuilder) {
+    if (!builder.data) {
+      return;
+    }
+
+    const dataFrames = builder.data.series.filter(
```

Comment:
Maybe out of scope but this seems rather simplistic. Could we maybe check the dataFrame shape? Cause not all data sources will return the `preferredVisualisationType` and the nodeGraph does not require it.

Reply:
Sounds good, I've updated the checks.